src/lib/db/project-store.ts

```diff
@@ -288,7 +288,31 @@ class ProjectStore implements IProjectStore {
         return members;
     }
 
-    async getMembersCountByProject(projectId?: string): Promise<number> {
+    async getProjectsByUser(userId: number): Promise<string[]> {
+        const members = await this.db.from((db) => {
+            db.select('project')
+                .from('role_user')
+                .leftJoin('roles', 'role_user.role_id', 'roles.id')
+                .where('type', 'root')
+                .where('name', 'Editor')
+                .where('user_id', userId)
```

Comment:
These are actually aliases in knex https://github.com/knex/knex/blob/8835d226529f8aebaa91c6ec77ccab47defef713/lib/query/querybuilder.js#L1706 but for semantics purposes, it might be more clear to use andWhere.